### ISSUE TITLE
How do the points work?

### ISSUE THREAD

**Issue by Muhammadibn** [imabusabah/perfectquran#10]
I have tried to work out how the points work. I was thinking they work according to how many letters there are or letters * 10 according to the hadith but it didn't match. 

Could you please enlighten me in this regard.

**Comment by imabusabah** [comment]
As salaam alaykum,

I had the idea of assigning points to different actions taken on the site, in order to incentivize learning. The implementation that you saw is an example of it poorly done. I have since removed it.

I thought about assigning points based on the hadith, which you are referring to, however, I don't have reliable data for the number of letters in each ayah.

Instead, I used the information that I generated to calculate percentages and then randomly multiplied them by 3.

So basically this:
```
# The bismala
Ayah.first.uthmani_text.length * 3 
#=> 114 

# ALM
Ayah.find(8).uthmani_text.length * 3 
#=> 15

# Ayat-ul-Kursi
Ayah.find(262).uthmani_text.length * 3 
#=> 1281
```
